spec/day_care_spec.rb

```diff
@@ -0,0 +1,61 @@
+require './lib/day_care'
+require './lib/customer'
+require './lib/pet'
+
+RSpec.describe DayCare do
+  before(:each) do
+    @day_care = DayCare.new("The Dog Spot")
+  end
+
+  it "exists" do
+    expect(@day_care).to be_instance_of(DayCare)
+  end
+
+  it "has a name" do
+    expect(@day_care.name).to eq("The Dog Spot")
+  end
+
+  it "starts with no customers" do
+    expect(@day_care.customers).to eq([])
+  end
+
+  it "can add customers" do
+    joel = Customer.new("Joel", 2)
+    billy = Customer.new("Billy", 3)
+
+    @day_care.add_customer(joel)
+    @day_care.add_customer(billy)
+    expect(@day_care.customers).to eq([joel, billy])
+  end
+
+  it "can list unfed pets" do
+    joel = Customer.new("Joel", 2)
+    samson = Pet.new({name: "Samson", type: :dog})
+    lucy = Pet.new({name: "Lucy", type: :cat})
+    joel.adopt(samson)
+    joel.adopt(lucy)
+    billy = Customer.new("Billy", 3)
+    molly = Pet.new({name: "Molly", type: :cat})
+    billy.adopt(molly)
+
+    @day_care.add_customer(joel)
+    @day_care.add_customer(billy)
+
+    lucy.feed
+
+    expect(@day_care.unfed_pets).to eq([samson, molly])
+
```

Comment:
Get rid of this blank space (47). Do we want to put these `Pets` and `Customers` in the before(:each)?

Reply:
If we put all of the objects in the `before(:each)` we would only need to add the `add_customer(joel)` in the tests that need it. Does that make sense?

Reply:
Just updated this - check out the latest changes.

Reply:
Love the additional `before(:each)` that @rtillies suggested. Thanks for amending this @cjsim89